backend/de.metas.cucumber/src/test/java/de/metas/cucumber/stepdefs/C_OrderLine_StepDef.java

```diff
@@ -449,10 +450,12 @@ public void metasfresh_contains_c_order_lines_expecting_error(@NonNull final Dat
 		try
 		{
 			metasfresh_contains_c_order_lines(dataTable);
+
+			Assertions.fail("An Exception should have been thrown !");
 		}
 		catch (final AdempiereException exception)
 		{
-			final String errorMsg = DataTableUtil.extractStringOrNullForColumnName(tableRows.get(0), "OPT.ErrorMessage");
+			final String errorMsg = DataTableUtil.extractStringOrNullForColumnName(tableRows.get(0), "ErrorMessage");
```

Comment:
The error-message is localized, so the string depends on the current language. better verify the error using AdempiereException.getErrorCode(). It contains the AD_Message.Value.
(just ported to this branch)